### ISSUE TITLE
Compilation process error

### ISSUE THREAD

**Issue by ChillsLove** [sam20908/vscode-fastolympiccoding#33]
Plugin version: Fast Olympic Coding v1.2.1
GCC version: gcc version 14.2.0 (x86_64-win32-seh-rev0, Built by MinGW-Builds project)
Error message:
```
D:/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot open output file "d:\WorkSpace\acm\test.exe": Invalid argument
collect2.exe: error: ld returned 1 exit status
```
configuration file:
[c_cpp_properties.json](https://github.com/user-attachments/files/18048176/c_cpp_properties.json)
[settings.json](https://github.com/user-attachments/files/18048177/settings.json)
[tasks.json](https://github.com/user-attachments/files/18048178/tasks.json)

Some of my attempts:
I tried to replace variables like ${...} with static absolute paths d:// Xxx.cpp or d: \ \Running xxx.exe through plugins is not a problem.
At the same time, I tested using cmd to compile gcc and found that using '\' in the path can compile normally, so I don't think the path in the error message will affect it

I hope you can help me solve my problem. If you could assist me, I would be extremely grateful.
I will continue to support this plugin. Previously, this plugin was only used on Sublime Text. Now that you have migrated it to VSCode, I would like to express my sincere gratitude to you.

**Comment by sam20908** [comment]
Can reproduce, fixing right now

**Comment by sam20908** [comment]
Fixed via dfa607285147f1cf272e9ec57b83cd5fcdf2ad86 which will be released on 1.2.2 in few minutes.
